emails/views.py

```diff
@@ -255,18 +255,29 @@ def _sns_message(message_json):
 
     text_content, html_content = _get_text_and_html_content(email_message)
 
+    # scramble alias so that clients don't recognize it and apply default link styles
+    display_email = re.sub('([@.:])', r'<span>\1</span>', to_address)
+    wrapped_html = render_to_string('emails/wrapped_email.html', {
+        'original_html': html_content,
+        'email_to': to_address,
+        'display_email': display_email,
+        'SITE_ORIGIN': settings.SITE_ORIGIN,
+    })
+
+    relay_from_address, relay_from_display = _generate_relay_From(from_address)
+
     ses_client = boto3.client('ses', region_name=settings.AWS_REGION)
     try:
         ses_response = ses_client.send_email(
             Destination={'ToAddresses': [relay_address.user.email]},
             Message={
                 'Body': {
-                    'Html': {'Charset': 'UTF-8', 'Data': html_content},
+                    'Html': {'Charset': 'UTF-8', 'Data': wrapped_html},
                     'Text': {'Charset': 'UTF-8', 'Data': text_content},
                 },
                 'Subject': {'Charset': 'UTF-8', 'Data': subject},
             },
-            Source=settings.RELAY_FROM_ADDRESS,
+            Source='{0} <{1}>'.format(relay_from_display, relay_from_address),
```

Comment:
Tested locally and found out the "display" from value needs to be inside `"` marks to work ...

```suggestion
            Source='"{0}" <{1}>'.format(relay_from_display, relay_from_address),
```

Reply:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/71928/83809029-62620280-a67b-11ea-9568-4c5c9c186bcc.png">

Reply:
Using `Address` from `email.headerregistry` properly formats with the `"`